packages/cbioportal-frontend-commons/package.json

```diff
@@ -18,15 +18,17 @@
   "license": "AGPL-3.0-or-later",
   "repository": "cBioPortal/cbioportal-frontend",
   "scripts": {
-    "build": "cross-env NODE_ENV=production NODE_OPTIONS=--max-old-space-size=2048 yarn run rollup",
+    "build": "yarn run tcm && cross-env NODE_ENV=production NODE_OPTIONS=--max-old-space-size=2048 yarn run rollup",
     "start": "yarn run watch",
-    "watch": "yarn run rollup:watch",
+    "watch": "concurrently \"yarn run tcm:watch\" \"yarn run rollup:watch\"",
     "watchSSL": "yarn run watch",
     "rollup": "rollup -c rollup.config.ts",
-    "rollup:watch": "rollup -c rollup.config.ts -w",
+    "rollup:watch": "yarn run rollup --watch",
+    "tcm": "tcm -p src/**/*.module.scss",
```

Comment:
Too bad we cannot add a comment in JSON. It took me a while to understand what tcm is. Maybe it's ok.

Reply:
Hmm, actually we can rename it to `typed-css` or something like that.